flybywiresim/a32nx
# Fix wing lights

Fixes #7614

## Summary of Changes
This PR fixes Wing light behavior as requested.

## Screenshots (if necessary)

## References

## Additional context
This bug appeared after having merged #7605 
This should be reopened.

**_IMPORTANT: SEE THE LAST TWO COMMITS ONLY_**. My repo is a mess due to multiple merge. Should be fixed once my other PRs merged.

## Testing instructions

1. Spawn on the runway
2. Set night time
3. Turn on and off Wing lights
   - [ ] Both sides should on/off


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
